src/app/src/pages/ForgotPassword.js

```diff
@@ -13,7 +14,10 @@ export default function ForgotPassword() {
 
     const formatApiError = apiErrorData => {
         var errorDetail = '';
-        Object.values(apiErrorData).forEach(val => (errorDetail += `${val}`));
+        for (const [field, errorArray] of Object.entries(apiErrorData)) {
```

Comment:
Since this is a common error delivery pattern by Django, consider pulling this `formatApiError` method into `utils.js`.